test/library/standard/Path/Prachi-Muskaan/testcases_expandUser.chpl

```diff
@@ -0,0 +1,10 @@
+/* Test cases to validate expandUser function in Path.chpl */
+
+use Path;
+
+writeln(expandUser('~user/'));
+writeln(expandUser('~USER/'));
```

Comment:
This test appears to check against a user name you know exists on the system and one you know doesn't.  It would be good to make those cases more explicit (i.e. comment this so we know what you're checking!), and more robust (if this is run on a system that just happens to have both `user` and `USER` as user names, then you aren't testing the same thing you would be testing on a system that only defines one, or defines neither).

Reply:
You always want to test:
- current user
- a user that exists on the system and is not the current user
- a user that explicitly does not exist